strings.txt

```diff
@@ -77,7 +80,7 @@ CREDITS
 	CS	Roman Smetak
 	DA	Ulrik Mehr, Morten Fischer-Madsen
 	DE	Melkon Torosyan, Thorsten Faltings, Michael Herger
-	ES	Andy Spiers, Néstor Spedalieri
+	ES	Andy Spiers, Néstor Spedalieri	
```

Comment:
Please make sure you don't change _any_ of the existing strings or languages accidentally.

Reply:
Oh gosh, my editor must have done that, I'll go through and check that